meetings/2021-09-08.md

```diff
@@ -0,0 +1,140 @@
+# 2021-09-08 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+## Agenda
+
+* Previous meeting minutes https://github.com/solid/authorization-panel/pull/254
+* **no access** by default for apps (clients)
+  * which use case is this related to?
+* [Access Modes](https://github.com/solid/authorization-panel/issues/253)
+
+## Present
+
+* Justin
+* Pavlik
+* Aaron
+* Henry
+* Sarven
+
+## Minutes
+
+### Previous meeting minutes
+
+Henry: I made PR but there were some unclear and missing parts. I asked few questions inline.
+**Sarven** I can add commit to that PR
+
+### No access by default for apps
+
+**PAVLIK** the perception that apps always have all of the access a given authenticated end-user has, but using client-id it is possible to restrict access except for allowed clients. Should the default be to block any clients but permitted ones. Related conversation https://github.com/solid/data-interoperability-panel/issues/139#issuecomment-915102197
+
+**Sarven** Maybe we should create issue first before we discuss things here.
+There is a discussion in WAC [issue 81: Client identification](https://github.com/solid/web-access-control-spec/issues/81) 
+
+**Henry** I think there is a thread on this in our discussion channel: on [proofs and Origins](
+https//github.com/solid/authorization-panel/discussions/223)
+
+ One thing that came out of that discussion, is that I don't think that this problem applies to [HTTPSig](https://github.com/solid/authentication-panel/blob/main/proposals/HttpSignature.md).  There a wallet is signing every request with the key of the user. The wallet, can thus be thought to be authorizing access to the resource independent of the client app, since it is signing a specific request. The wallet is responsible for enforcing user authorization on the client, by choosing which resources it will sign requrests for. 
+
+**Justin** We have 2 dimensions to consider: a) the social entity (eg. me) and b) the application used (by me). Should we discuss it here? Do I want to restrict access to only a subset of application which I use?
+I would say yes, there is a benefit.
+Already work on Solid-OIDC provides us with `client_id`. HTTPSig can deal with this issue in a different way.
+There seems to be notion that as soon as app gets a token, it can access anything that user can access.
+We don't want to create such perception. As soon as we review draft for ACP or WAC+ or something else.
+We will evaluate it on how it gives restrictions on clients.
+
+**Henry** Note with HttpSig the Wallet can have policies about which apps it gives signtatures to for what sets of resources, so that access control does not need to be done in server ACRs.
```

Comment:
```suggestion
**Henry** Note that with HttpSig, the Wallet can have policies about which apps it gives signatures to for what sets of resources, so that access control does not need to be done in server ACRs.
```